cmd/testbucket/golist.go

```
@@ -0,0 +1,305 @@
+			return "", fmt.Errorf("no go.work or .git found above %s", dir)
+		}
+		cur = parent
+	}
+}
+
+// workspaceMembers reads go.work through `go work edit -json` rather than
+// parsing it: the file in this repo is mostly a long rationale comment, and
+// the toolchain is the only correct parser of the directive it wraps.
+func workspaceMembers(repoRoot string) (map[string]bool, error) {
+	if _, err := os.Stat(filepath.Join(repoRoot, "go.work")); err != nil {
+		return map[string]bool{}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 65) but it returns nil

(nilerr)